sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLCommandSuite.scala

```diff
@@ -694,19 +694,6 @@ class DDLCommandSuite extends PlanTest {
     assertUnsupported("ALTER TABLE table_name SKEWED BY (key) ON (1,5,6) STORED AS DIRECTORIES")
   }
 
-  test("alter table: change column name/type/position/comment (not allowed)") {
```

Comment:
The test suite `DDLCommandSuite` is used for testing the parser. Could you added unit test cases for the new parser rules?